### ISSUE TITLE
Question about infoNCE loss 

### ISSUE THREAD

**Issue by diegodibe** [cruiseresearchgroup/TSCP2#7]
Thank you for your work. 
I have hard times understanding the following two lines in the loss function. 

In fact, taking out the sum in the numerator of _logits_(performed over the batch dimension)
https://github.com/cruiseresearchgroup/TSCP2/blob/f353f09657f0ba73aec98d0cdeee0b8cc78fe9d3/src/losses.py#L100
and computing the -log of _logits_ instead of:
https://github.com/cruiseresearchgroup/TSCP2/blob/f353f09657f0ba73aec98d0cdeee0b8cc78fe9d3/src/losses.py#L104
It would look like the formula of the InfoNCE: 
![image](https://user-images.githubusercontent.com/40866333/215797627-504e5a3a-ce49-460a-9fa3-f930594b884c.png)

Could you please provide an intuition of your implementation?
Thank you in advance
